packages/common/src/core/schemas/internal/ConfigurableEntity.ts

```diff
@@ -2,3 +2,8 @@
  * Generic interface for a Hub Entity
  */
 export type ConfigurableEntity = Record<string, any>;
```

Comment:
I don't think this `type` accurately represents what we'd expect. I think we should make it more explicit: 

```ts
export type ConfigurableEntity = HubEntity & {};
```

And if there are additional properties we need to pass in with the entity, we can add them to the intersection here. 

This change would require small tweaks in the `getFeaturedImageUrl`, `getLocationOptions`, and `getLocationExtent` internal utils, but I think that's okay. I think you could do one of two things: 

```ts
// getFeaturedImageUrl
export function getFeaturedImageUrl(
  entity: Exclude<ConfigurableEntity, IHubGroup>, // fixes the type error
  ...
) {...}
```
OR

```ts
// getFeaturedImageUrl
export function getFeaturedImageUrl(
  entity: ConfigurableEntity,
  ...
) {
  const featuredImageUrl = getProp(entity, "view.featuredImageUrl") // getProp avoids the type error
  ...
}
```